sirepo/pkcli/job_agent.py

```diff
@@ -332,11 +333,9 @@ async def _fastcgi_op(self, msg):
         if not self.fastcgi_cmd:
             m = msg.copy()
             m.jobCmd = 'fastcgi'
-            self._fastcgi_file = 'job_cmd_fastcgi.sock'
+            self._fastcgi_file = str(pkio.py_path(cfg.fastcgi_sock_dir).join('job_cmd_fastcgi.sock'))
```

Comment:
Since this is now in a public directory, it needs to be unique. This won't matter when running inside docker, but it will matter when running locally and running the tests and on sbatch. Something like this:

`cfg.fastcgi_sock_dir.join(f'sirepo_job_cmd-{cfg.agent_id:8}.sock')`

The only place `str()` is needed is on the bind_unix_socket. I shortened the name since we have to give more context. 8 chars of the agent id are enough in this context. Could use something else, but this would help with debugging in local environment.

Reply:
https://github.com/radiasoft/sirepo/pull/2773/commits/56c0225deec58e83d9eb23d034a73a052299b95a